ios/Mattermost.xcodeproj/project.pbxproj

```diff
@@ -582,9 +602,11 @@
 				TargetAttributes = {
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = UQ8HT4Q2XM;
```

Comment:
Did we mean to change `DevelopmentTeam` here? I'm sure xcode set it automatically, but I don't think it's necessary. We've been getting by just fine with only declaring `DEVELOPMENT_TEAM` in `Info.plist`. Now there will be a couple more places I have to change if I'm gonna run this on my own device. (I have a stash with my own signing identity declarations that I have to reapply every time I wanna run on an iOS device.)